packages/ui/components/avatar/UserAvatarGroup.tsx

```diff
@@ -8,24 +10,42 @@ import { AvatarGroup } from "./AvatarGroup";
 type UserAvatarProps = Omit<React.ComponentProps<typeof AvatarGroup>, "items"> & {
   users: (Pick<User, "name" | "username" | "avatarUrl"> & {
     profile: Omit<UserProfile, "upId">;
-    hideTruncatedAvatarsCount?: boolean;
```

Comment:
Wrong. Items of `users` do not have `hideTruncatedAvatarsCount` prop. This prop rather exists in `React.ComponentProps<typeof AvatarGroup>`